src/solvers/dgmulti/dg_parabolic.jl

```diff
@@ -107,7 +110,9 @@ function calc_gradient_volume_integral!(gradients, u, mesh::DGMultiMesh{NDIMS, <
 
     # rotate to physical frame on each element
     for i in eachdim(mesh), j in eachdim(mesh)
-      @. gradients[i][:, e] = gradients[i][:, e] + dxidxhatj[i, j][:, e] * local_reference_gradients[j]
+      for node in eachindex(local_reference_gradients[j])
+        gradients[i][node, e] = gradients[i][node, e] + dxidxhatj[i, j][node, e] * local_reference_gradients[j][node]
+      end
```

Comment:
It should also be possible to use views here, e.g., `@views @. ...`. This should give you the same performance without writing a loop.